researchapp/fixtures.yml

```diff
@@ -0,0 +1,39 @@
+---
+class: !!python/name:researchapp.models.participants.Participant
+args: {}
+---
+class: !!python/name:researchapp.models.providers.Organization
+args:
+  id: smart
+  name: Smart
+---
+class: !!python/name:researchapp.models.providers.Organization
+args:
+  id: allscripts
+  name: Allscripts
+---
+class: !!python/name:researchapp.models.providers.Api
+args:
+  url: http://smart.dev.syncfor.science:9000/api/fhir/
```

Comment:
My reasoning for putting the URL on the "Api" object was based on our conversation yesterday about how there isn't really a place for URLs in the FHIR Organization model. My hope was that Organization and Practitioner would be able to follow the FHIR spec pretty closely.